src/ui/map.js

```diff
@@ -798,6 +798,9 @@ class Map extends Camera {
      * const bounds = map.getBounds();
      */
     getBounds(): LngLatBounds | null {
+        if (this.transform.projection.name === 'globe') {
+            warnOnce('Globe projection does not support getBounds API, this API may behave unexpectedly."');
```

Comment:
Nit: should we consider being more specific here?
```suggestion
            warnOnce('Globe projection does not support getBounds API, this API may behave unexpectedly when viewing the poles."');
```

Reply:
Good catch, thanks @SnailBones 👍